docs/source/Usage.rst

```diff
@@ -29,13 +39,13 @@ Version >= 3.9
 
 We recommend using python virtual environments for the usage of FABulous.
 If you are not sure what this is and why you should use it, please read the
-:`virtualenv documentation <https://virtualenv.pypa.io/en/latest/index.html>`_:
+`virtualenv documentation <https://virtualenv.pypa.io/en/latest/index.html>`_.
 
 .. code-block:: console
 
    $ cd FABulous
-    $ virtualenv venv
-    $ source venv/bin/activate
+   $ virtualenv venv
```

Comment:
Does the copy code block also copy the `$`? Sometimes, it is annoying when copying a command and getting an invalid command because of the `$` sign.  
Some references might help solve the problem: 
https://github.com/executablebooks/sphinx-book-theme/issues/411
https://sphinx-copybutton.readthedocs.io/en/latest/use.html#using-regexp-prompt-identifiers

Reply:
Until now we don't use the copybutton feature. Should I add it, along with the suggested configuration? I tried it out and it worked perfectly. Using the copybutton without the configuration does indeed also copy the `$` as you suggested. I think it would be a useful feature to have a copy button, but then we would also have to adjust the config so that `(venv)` and `(FABulous>)` are also ignored.

Reply:
You can decide including it in this pull request or leave it later. The adjustment should be simple, is just some extra regex need to match.

Reply:
I just added `|FABulous\> |\(venv\)\$` to the configuration to also ignore `(venv)` and `FABulous>` (but not `FABulous`!). Maybe there's a better/smarter way, but it worked. 

One thing that didn't work was ignoring comments starting with `#` which we have e.g. [here](https://github.com/FPGA-Research-Manchester/FABulous/blob/FABulous2.0-development/docs/source/FPGA-to-bitstream/Yosys%20compilation.rst?plain=1#L25). I tried to add `|^\#.*` which does work on https://regex101.com/ but in the actual documentation I had no success.

Reply:
Maybe try doing as raw string. 

The `^` means start of the line. The line it try to match might have space, which make it not start with `#`, I think without the `#` should work.

Reply:
> Maybe try doing as raw string.

You mean with r""? I already did that, since they already suggest it in the sphinx docs.

 After trying out a lot of different stuff, I found out that the comment in `Usage.rst` is automatically ignored without explicitely mentioning it in the regex. So I just added the 'FABulous> ` prefix to the other comment and then it also worked there. I will push the changes later.